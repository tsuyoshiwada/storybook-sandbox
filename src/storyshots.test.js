import initStoryshots, { snapshotWithOptions } from '@storybook/addon-storyshots';
import path from 'path';

initStoryshots({
  framework: 'react',
  configPath: path.join(__dirname, '..', '.storybook'),
});
