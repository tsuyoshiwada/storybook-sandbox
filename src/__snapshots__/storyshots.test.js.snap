// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button with text 1`] = `
<button
  className="sc-bdVaJa HcyVC"
  type="button"
>
  Primary Button
</button>
`;
